config-model/src/main/java/com/yahoo/vespa/model/admin/clustercontroller/ClusterControllerContainer.java

```diff
@@ -79,7 +79,7 @@ public int getWantedPort() {
 
     @Override
     public boolean requiresWantedPort() {
-        return index() == 0;
+        return false;
```

Comment:
Would lead to collision if multiple CCs on same host, from different clusters. This isn't something we support, anyway. If there are overlapping CCs, they're joined. Otherwise they're disjoint, or there's a single cluster.